poollet/machinepoollet/controllers/machinepool_controller_test.go

```diff
@@ -27,12 +29,113 @@ import (
 	. "sigs.k8s.io/controller-runtime/pkg/envtest/komega"
 )
 
-const (
-	TestMachinePool = "test-machinepool"
-)
-
 var _ = Describe("MachinePoolController", func() {
-	_, _, _, srv := SetupTest()
+	ns, mp, mc, srv := SetupTest()
```

Comment:
please don't abbreviate machine pool / machine class that much - makes it hard to read for first-time readers.
Applies for all occurrences in this file.